db/migrate/20210119005647_add_created_via_to_incoming_email.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCreatedViaToIncomingEmail < ActiveRecord::Migration[6.0]
+  def change
+    add_column :incoming_emails, :created_via, :integer, null: true
```

Comment:
This column kind of models how an enum column would work. I was wondering if we should just introduce an `unknown` type to the existing rows so that we can change this to `null: false`. Otherwise, I worry that we may add new code in the future that creates the `IncomingEmail` record without knowing that `created_via` is a required column.

Reply:
Oh good point. If we can avoid `NULL` we absolutely should.